trigger/task.go

```diff
@@ -355,7 +355,7 @@ func (t *taskTriggers) generate(sub *event.Subscription, pastTenseOverride, test
 					return nil, errors.Wrapf(err, "error getting execution task")
 				}
 				if executionTask.Details.Status == evergreen.TaskFailed {
-					if executionTask.Details.Description == evergreen.TaskDescriptionStranded {
+					if executionTask.Details.Description == evergreen.TaskDescriptionStranded || executionTask.GetDisplayStatus() == evergreen.TaskSetupFailed {
```

Comment:
GetDisplayStatus does a lot of work which we don't actually need to do; this could instead be handled by just checking `t.Details.Type == evergreen.CommandTypeSetup` (which is the step inside GetDisplayStatus that returns task setup failed). This is how we handle this elsewhere for task triggers https://github.com/evergreen-ci/evergreen/blob/c2fc7132326fa66cf2be16bcc719fc934580cb06/trigger/task.go#L475.
We should probably also skip the ticket if IsSystemUnresponsive is true.